plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -1050,13 +1055,34 @@ coverageCompile originalExpr exprType src compiledTerm covT =
     findHeadSymbol (GHC.Cast t _) = findHeadSymbol t
     findHeadSymbol _              = Nothing
 
+-- | We cannot compile the unfolding of `GHC.Num.Integer.integerNegate`, so we directly
```

Comment:
and why does that matter?
```suggestion
-- | We cannot compile the unfolding of `GHC.Num.Integer.integerNegate`, which is important because GHC inserts calls to it when it sees negations, even negations of literals (unless NegativeLiterals is on, which it usually isn't). So we directly
```